plugins/ocm/README.md

```diff
@@ -1,104 +1,114 @@
+The Open Cluster Management (OCM) plugin integrates your Backstage instance with the `MultiClusterHub` and `MultiCluster` engines of Open Cluster Management.
 
 ## Prerequisites
 
-1. OCM deployed and configured on a kubernetes cluster
-2. Installed [Kubernetes plugin for Backstage](https://backstage.io/docs/features/kubernetes/overview)
-3. Kubernetes plugin is properly configured and able to connect to the Hub cluster for OCM via a `ServiceAccount` (the cluster is accessed by backstage backend, therefore it requires a permanent connection with constant RBAC scope)
-4. Following `ClusterRole` must be granted to `ServiceAccount` accessing the Hub cluster:
-   ```yaml
-   kind: ClusterRole
-   apiVersion: rbac.authorization.k8s.io/v1
-   metadata:
-     name: backstage-ocm-plugin
-   rules:
-     - apiGroups:
-         - cluster.open-cluster-management.io
-       resources:
-         - managedclusters
-       verbs:
-         - get
-         - watch
-         - list
-     - apiGroups:
-         - internal.open-cluster-management.io
-       resources:
-         - managedclusterinfos
-       verbs:
-         - get
-         - watch
-         - list
-   ```
+- OCM is deployed and configured on a Kubernetes cluster.
+- [Kubernetes plugin for Backstage](https://backstage.io/docs/features/kubernetes/overview) is installed.
+- Kubernetes plugin is configured and able to connect to the Hub cluster for OCM using a `ServiceAccount`. The cluster is accessed using Backstage backend, therefore, the cluster requires a permanent connection with constant RBAC scope.
+- The `ClusterRole` must be granted to `ServiceAccount` accessing the Hub cluster as follows:
+  ```yaml
+  kind: ClusterRole
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: backstage-ocm-plugin
+  rules:
+    - apiGroups:
+        - cluster.open-cluster-management.io
+      resources:
+        - managedclusters
+      verbs:
+        - get
+        - watch
+        - list
+    - apiGroups:
+        - internal.open-cluster-management.io
+      resources:
+        - managedclusterinfos
+      verbs:
+        - get
+        - watch
+        - list
+  ```
 
 ## Capabilities
 
-- All clusters represented as `ManagedCluster`s in MultiClusterHub/MCE are discovered and imported into Backstage catalog:
+The OCM plugin has the following capabilities:
+
+- All clusters represented as `ManagedCluster` in `MultiClusterHub` or MCE are discovered and imported into the Backstage catalog, such as:
   - Entity is defined as `kind: Resource` with `spec.type` set to `kubernetes-cluster`.
-  - Links to OCP console, OCM console and OpenShift Cluster Manager are provided in `metadata.links`.
-- Show real time data from OCM on Resource entity page including:
-  - Cluster current status (up, down, etc.)
-  - Cluster details (console link, OCP and Kubernetes version, etc.)
+  - Links to the OCP console, OCM console, and OpenShift Cluster Manager are provided in `metadata.links`.
```

Comment:
```suggestion
  - Links to the OpenShift Container Platform console, OCM console, and OCM are provided in `metadata.links`.
```

Reply:
Check on official usage of OpenShift naming maybe? Not sure how we want to refer to this in upstream docs like this